.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -84,9 +112,16 @@ jobs:
       - uses: ./.github/actions/prepare-metal-run
         with:
           arch: ${{ matrix.test-group.arch }}
-      - name: Run ttnn sweeps
+      - name: Setup ttnn sweeps runner environment
         run: |
           source ${{ github.workspace }}/python_env/bin/activate
           cd $TT_METAL_HOME
           export PYTHONPATH=$TT_METAL_HOME
+      - name: Run ttnn sweeps (single sweep)
+        if: ${{ github.event.inputs.sweep_name != '' && github.event.inputs.sweep_name != 'ALL SWEEPS' }}
+        run: |
+          python tests/sweep_framework/runner.py --module-name ${{ github.event.inputs.sweep_name }} --elastic cloud --tag ci-main
+      - name: Run ttnn sweeps (all sweeps)
+        if: ${{ github.event.inputs.sweep_name == '' || github.event.inputs.sweep_name == 'ALL SWEEPS' }}
```

Comment:
Why not make a default value as `ALL_SWEEPS`?

Reply:
When we schedule this is there a way to pass that value? I set it up this way so if it's a scheduled run, it won't have an input and will run all, or if `ALL SWEEPS` is selected.

Reply:
Hmm, actually, how about making that the default value, but you could also use `github.event.name == 'schedule'` to your advantage.

Reply:
Done